doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
+<p align="center"><b>Table 2-1:</b> Containerised Platform: General Requirements.</p>
+
+<a name="2.3.2"></a>
+### 2.3.2 Infrastructure
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.inf.com.01` | Compute | The Architecture **must** provide compute resources for Pods.  |
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). |
+| `req.inf.stg.01` | Storage | The Architecture **must** provide shared Block storage for Containers.
+| `req.inf.stg.02` | Storage | The Architecture **must** provide shared Object storage for Containers.
+| `req.inf.stg.03` | Storage | The Architecture **may** provide local file system storage solution for Containers.
+| `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.09` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) CNF/VNF workloads. |
+| `req.inf.ntw.10` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.11` | Network | The SDN solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.12` | Network | The SDN solution **should** be able to be centrally administrated and configured. |
```

Comment:
For me it is unclear whether we are mandating an SDN solution by placing a **should** requirement on it's behavior. For me having (or not having) an SDN solution are means of implementing the required networking capabilities, not the requirement itself.

Perhaps substituting "The SDN solution" with "The networking solution" would be a better way of phrasing this.